zerver/data_import/rocketchat.py

```diff
@@ -0,0 +1,673 @@
+import logging
+import os
+import subprocess
+from typing import Any, Dict, List, Set
+
+import bson
+from django.conf import settings
+from django.forms.models import model_to_dict
+from django.utils.timezone import now as timezone_now
+
+from zerver.data_import.import_util import (
+    SubscriberHandler,
+    ZerverFieldsT,
+    build_message,
+    build_personal_subscriptions,
+    build_realm,
+    build_recipients,
+    build_stream,
+    build_stream_subscriptions,
+    build_user_profile,
+    build_zerver_realm,
+    create_converted_data_files,
+    make_subscriber_map,
+    make_user_messages,
+)
+from zerver.data_import.sequencer import NEXT_ID, IdMapper
+from zerver.data_import.user_handler import UserHandler
+from zerver.lib.emoji import name_to_codepoint
+from zerver.lib.utils import process_list_in_batches
+from zerver.models import Reaction, Recipient, UserProfile
+
+
+def make_realm(realm_id: int, realm_subdomain: str, domain_name: str) -> ZerverFieldsT:
+    NOW = float(timezone_now().timestamp())
```

Comment:
If the Rocket.Chat export has a realm creation time, it's possible we should record that instead of the current time.  

(This may be an issue across our data import tools)

Reply:
Mattermost does not provide any timestamps (realm creation and stream creation) and Slack only provides the stream creation timestamp, which is been correctly used while creating Zulip streams. So, this is a problem with Zulip only.

Will fix it.

Reply:
Update code to use the actual realm creation and stream creation time.